### ISSUE TITLE
Package unmaintained

### ISSUE THREAD

**Issue by Ergus** [coldnew/linum-relative#48]

**Comment by abo-abo** [comment]
I found this package by keyword "converience". I suggest to fix the typo when the package becomes maintained :)

**Comment by Ergus** [comment]
I don't think it will become maintained again as display-line-numbers actually offers all the functionalities.. and some more added recently
